test/encoder.js

```diff
@@ -1,129 +1,252 @@
-import 'babel-polyfill';
 import { assert } from 'chai';
+import BN from 'bn.js';
+
 import Encoder from '../src/encoder';
 
 describe('Encoder', function() {
+  const UINT256_MAX = '115792089237316195423570985008687907853269984665640564039457584007913129639935';
+  const UINT256_MIN = '0';
+  const INT256_MAX = '57896044618658097711785492504343953926634992332820282019728792003956564819967';
+  const INT256_MIN = '-57896044618658097711785492504343953926634992332820282019728792003956564819968';
 
   describe('getFunctionHash()', function() {
-    const funcObj = {"constant": false,"inputs": [{"name": "_resultIndex","type": "uint8"},{"name": "_sender","type": "address"},{"name": "_amount","type": "uint256"}],"name": "voteFromOracle","outputs": [{"name": "","type": "bool"}],"payable": false,"stateMutability": "nonpayable","type": "function"};
-
-    it('should convert a function obj to hash string', async function() {
-      const hash = await Encoder.getFunctionHash(funcObj);
+    const funcObj = {
+      "constant": false,
+      "inputs": [{
+        "name": "_resultIndex",
+        "type": "uint8"
+      }, {
+        "name": "_sender",
+        "type": "address"
+      }, {
+        "name": "_amount",
+        "type": "uint256"
+      }],
+      "name": "voteFromOracle",
+      "outputs": [{
+        "name": "",
+        "type": "bool"
+      }],
+      "payable": false,
+      "stateMutability": "nonpayable",
+      "type": "function"
+    };
+
+    it('should convert a function obj to hash string', function() {
+      const hash = Encoder.getFunctionHash(funcObj);
       assert.equal(hash, '006a8a32');
       assert.equal(hash.length, 8);
     });
 
-    it('throws if methodObj is undefined', async function() {
```

Comment:
Yeah, don't think "async" keyword is needed

Reply:
:+1: erased all unnecessary async